run_dir/static/js/worksets.js

```diff
@@ -37,20 +37,33 @@ function load_table() {
       size++;
       var tbl_row = $('<tr>');
       $.each(columns, function(i, column_tuple){
+          var content='';
+          if (column_tuple[1] == 'projects'){
+            $.each(summary_row['projects'], function(project_id, project_data ){
+                content+='<a href="/project/' + project_id + '">' + project_id + '</a> (' + project_data['samples_nb'] + ')  ';
+            });
+          }else if (column_tuple[1] == 'application' || column_tuple[1] == 'library_method'){
+              content = summary_row[column_tuple[1]].join(', ');
+          }else if (column_tuple[1] == 'passed' || column_tuple[1] == 'failed' || column_tuple[1] == 'unknown' || column_tuple[1] == 'total'){
```

Comment:
That might make the code harder to understand though, and it is already pretty dense already :D